external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -89,7 +90,9 @@ class TrainConfig:
     test_url: str
     out_url: str
     transform: TransformConfig = field(default_factory=TransformConfig)
-    tensor_transform: List[TransformedVariableConfig] = field(default_factory=list)
+    tensor_transform: List[
+        Union[TransformedVariableConfig, ConditionallyScaled]
```

Comment:
Maybe `Union[TransformedVariableConfig, ConditionallyScaled]` can be defined outside of this class as `Transforms` or something.

Reply:
If they start getting used elsewhere sure, but this Union is only used for dacite really. I used the `TransformFactory` protocol  for type-checking most places.